sources/packages/backend/apps/db-migrations/src/sql/Reports/Create-data-inventory-report.sql

```diff
@@ -0,0 +1,507 @@
+            jsonb_array_length(applications.data -> ''dependants'') as "Number of dependant children in PSE",
+            ''--TBD--'' as "Number of Dep (0-11 y.o. and disabled 12 y.o. or older)",
+            ''--TBD--'' as "Number of Dep (12 y.o.+ NOT disabled) and Other Dep Relatives",
+            applications.data ->> ''supportnocustodyDependants'' as "Has dependants without sole custody flag",
+            -- Institution Info
+            institution_locations.institution_code as "Institution Code (location code)",
+            ''--TBD--'' as "Alphabetical Institution Listing",
+            ''--TBD--'' as "Institution Location",
+            ''--TBD--'' as "Institution Type",
+            ''--TBD--'' as "Institution Contact List",
+            ''--TBD--'' as "De-designation",
+            -- Program Info
+            education_programs.program_name as "Program name",
+            education_programs.credential_type as "Credential type",
+            education_programs.cip_code as "Classification of Instructional Program (CIP) Code",
+            education_programs.field_of_study_code as "Field of Study Code",
+            education_programs.noc_code as "National Occupation Classification",
+            education_programs.sabc_code as "SABC Program Code",
+            education_programs.institution_program_code as "Institution Program Code",
+            (
+                case
+                    when education_programs.program_intensity = ''Full Time and Part Time'' then ''Y''
+                    else ''N''
+                end
+            ) as "Part-Time eligible (Yes/No)",
+            ''--TBD--'' as "Program Delivery ",
+            education_programs.completion_years as "Program Length",
+            ''Credit or Hours Based'' as "Credit or Hours Based",
+            ''--TBD--'' as "Program Regulatory Body",
+            ''--TBD--'' as "Entrance Requirements",
+            ''--TBD--'' as "Program ESL Content Percentage",
+            ''--TBD--'' as "Program offered jointly/ in partnership (Yes/No)",
+            ''--TBD--'' as "Program Work Integrated Learning (WIL) Component (Yes/No)",
+            ''--TBD--'' as "WIL regulatory approval (Yes/No � as applicable)",
+            ''--TBD--'' as "Program Field Trip / Travel Component (Yes/No)",
+            ''--TBD--'' as "Program international exchange component (Yes/No)",
+            ''--TBD--'' as "Program was pending but then aproved",
+            -- Study Period Offering Info
+            education_programs_offerings.offering_name as "Study Period Offering Name",
+            education_programs_offerings.year_of_study as "Year of Study",
+            education_programs_offerings.offering_intensity as "Offering intensity",
+            ''--TBD--'' as "Offering course load percentage",
+            education_programs_offerings.offering_delivered as "Offering delivery method",
+            ''--TBD--'' as "Offering WIL component (Yes/No)",
+            to_char(
```

Comment:
Similar to the `students.birth_date` is there a reason to not return these columns' raw value instead of using `to_char`?

Reply:
I just thought that the time part of the date is not useful for those date values.

Reply:
Those columns are data only and do not contain time (same as the mentioned `students.birth_date`) unless I am missing something.